ui/entry.jsx

```diff
@@ -0,0 +1,6 @@
+var React = require('react'),
+    Report = require('./components/report.jsx');
+
+require('./styles/base.css');
+
+React.render(<Report data={data}/>, document.getElementById('report'));
```

Comment:
1. You need to tell ESLint to lint `.jsx` files as well. You do this by calling `eslint --ext js --ext jsx`.
2. ESLint will tell you that `data` is undefined. Tell it it's a global using `/* globals data */`.
3. You can get the name of the variable by using

```js
var reportGenerator = require('lib/report-generator.js'),
    data = window[reportGenerator.MOCHA_TEST_DATA_VAR_NAME];
```
4. You might not need the globals pragma anymore if you access the variable from `window`.